sno/working_copy/sqlserver_adapter.py

```diff
@@ -0,0 +1,179 @@
+from sno import crs_util
+from sno.schema import Schema, ColumnSchema
+
+from sqlalchemy.sql.compiler import IdentifierPreparer
+from sqlalchemy.dialects.mssql.base import MSDialect
+
+
+_PREPARER = IdentifierPreparer(MSDialect())
+
+
+def quote(ident):
+    return _PREPARER.quote(ident)
```

Comment:
We need to drop these for SQLAlchemy's rather than writing more :)

Reply:
This is sqlalchemy's quote. It's just easier than writing 
IdentifierPreparer(MSDialect()).quote(...) everytime.

However, you are right that I shouldn't be quoting as many things manually. The quote() in this file could go away if i generated a proper sqlalchemy Table definition for a dataset and create that, instead of generating appropriately quoted SQL. The amount of work is about the same - just haven't switched it yet. The other obstacle is it might make the code less symmetrical in terms of creating tables and reading table structures - not sure until I try it.

Reply:
This is sqlalchemy's quote - I just wrapped it since writing `quote()` is easier than writing `IdentifierPreparer(MSDialect()).quote()`

But you are right - I should be trying to avoid manually quoting things. This quote here is only called when generating a table spec eg `("FID" bigint, "GEOM" geography, "CRT_DATE" datetime)` - it could go away if, instead of generating a table specification in SQL, I created one using sqlalchemy types and created that. It's about the same amount of work, but I just haven't switched to it yet. (It also might not end up being as symmetrical as the current code with regards to writing table specs and reading table specs - I don't know exactly unless I try to write it).

Reply:
One option is to use use `Table()`/etc, but you can also use the SA SQL expression builder which has escaping functions/templating/etc for SQL strings.

Reply:
Yes - will revisit this soon to try and get rid of lots of f-strings in all 3 working copies.